youtube_dl/extractor/indavideo.py

```diff
@@ -0,0 +1,87 @@
+                    {'url': '//pics.indavideo.hu/videos/001/837/039/3-1.jpg', 'id': '2'},
+                    {'url': '//pics.indavideo.hu/videos/001/837/039/4-1.jpg', 'id': '3'},
+                    {'url': '//pics.indavideo.hu/videos/001/837/039/5-1.jpg', 'id': '4'},
+                    {'url': '//pics.indavideo.hu/videos/001/837/039/6-1.jpg', 'id': '5'},
+                    {'url': '//pics.indavideo.hu/videos/001/837/039/7-1.jpg', 'id': '6'},
+                ],
+                'description': '',
+                'uploader': 'cukiajanlo',
+                'uploader_id': '83729',
+                'duration': 72,
+                'age_limit': 0,
+                'tags': ['tánc', 'cica', 'cuki', 'cukiajanlo', 'newsroom']
+            },
+        },
+        {
+            'url': 'http://indavideo.hu/video/Vicces_cica_1',
+            'md5': '8c82244ba85d2a2310275b318eb51eac',
+            'info_dict': {
+                'id': '1335611',
+                'title': 'Vicces cica',
+                'url': 'http://main-director.indavideo.hu/indavideo/911/001/335/611/Funny_Cat_back_Kick.360.mp4',
+                'ext': 'mp4',
+                'display_id': 'Vicces_cica_1',
+                'thumbnails': [
+                    {'url': '//pics.indavideo.hu/videos/001/335/611/1-1.jpg', 'id': '0'},
+                    {'url': '//pics.indavideo.hu/videos/001/335/611/2-1.jpg', 'id': '1'},
+                    {'url': '//pics.indavideo.hu/videos/001/335/611/3-1.jpg', 'id': '2'},
+                    {'url': '//pics.indavideo.hu/videos/001/335/611/4-1.jpg', 'id': '3'},
+                    {'url': '//pics.indavideo.hu/videos/001/335/611/5-1.jpg', 'id': '4'},
+                    {'url': '//pics.indavideo.hu/videos/001/335/611/6-1.jpg', 'id': '5'},
+                ],
+                'description': 'Játszik a tablettel. :D',
+                'uploader': 'Jet_Pack',
+                'uploader_id': '491217',
+                'duration': 7,
+                'age_limit': 0,
+                'tags': ['vicces', 'macska', 'cica', 'ügyes', 'nevetés', 'játszik', 'Cukiság', 'Jet_Pack'],
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        video_disp_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_disp_id)
+
+        embed_url = self._html_search_regex(r'<link rel="video_src" href="(.+?)"/>', webpage, 'embed_url')
+        video_hash = embed_url.split('/')[-1]
+
+        payload = self._download_json('http://amfphp.indavideo.hu/SYm0json.php/player.playerHandler.getVideoData/' + video_hash, video_disp_id)
+        video_info = payload['data']
+
+        return {
+            'id': video_info['id'],
+            'title': video_info['title'],
+            'url': video_info['video_file'],
+            'ext': 'mp4',
+            'display_id': video_disp_id,
+            'thumbnails': [{'url': x} for x in video_info['thumbnails']],
+            'description': video_info['description'],
```

Comment:
Every field apart from `title` and `url` should be treated optional and shouldn't break extraction if missing. Change every other field to `video_info.get('description')`